src/ui/Facet/BreadcrumbValuesList.ts

```diff
@@ -45,66 +45,66 @@ export class BreadcrumbValueList {
     if (this.elem) {
       return (
         `${this.facet.options.title}: ` +
-        map($$(this.elem).findAll('.coveo-facet-breadcrumb-value'), (value: HTMLElement) => {
-          return $$(value).text();
-        }).join(', ')
+        $$(this.elem)
+          .findAll('.coveo-facet-breadcrumb-value')
+          .map((value: HTMLElement) => {
+            return $$(value).text();
+          })
+          .join(', ')
       );
     }
     return '';
   }
 
   private buildExpanded() {
-    each(this.expanded, (value: FacetValue, index?: number) => {
+    this.expanded.forEach((value: FacetValue) => {
       const elementBreadcrumb = new this.breadcrumbValueElementKlass(this.facet, value).build();
       this.valueContainer.appendChild(elementBreadcrumb.el);
     });
   }
 
   private buildCollapsed() {
-    const numberOfSelected = filter(this.collapsed, (value: FacetValue) => value.selected).length;
-    const numberOfExcluded = filter(this.collapsed, (value: FacetValue) => value.excluded).length;
+    const numberOfSelected = this.collapsed.filter((value: FacetValue) => value.selected).length;
```

Comment:
Possible problem from moving from underscore.filter to array.filter : 

If `this.collapsed` can ever be null or undefined (because search-ui project does not have strict null check enabled), this would now throw null ref at run time.

Versus underscore that could (I think?, not sure 🤷  ) instead return an empty array.